### ISSUE TITLE
Enums and bitmasks

### ISSUE THREAD

**Issue by PMeira** [dss-extensions/OpenDSSDirect.jl#76]
A few enumerations are used in bitmasks or mixed enum+mask. Currently, `CEnum` doesn't allow this.

An interim fix was added for `Monitors.Mode` in https://github.com/dss-extensions/OpenDSSDirect.jl/pull/75 (just allow integers without restricting to the enum values), but a more elegant solution would be better. I also noticed a few projects vendor CEnum and modify it, for example https://github.com/JuliaGPU/VulkanCore.jl/pull/17/files

Looking for alternative packages, so far I found https://github.com/jmert/BitFlags.jl 
Should we adopt it here?

**Comment by kdheepak** [comment]
Yeah, I think generally we can do a lot to make a more Julian interface here.

**Comment by kdheepak** [comment]
I will look into those packages. I seem to remember solving this for work differently. When I’m back at my work computer I’ll try to find that solution and paste it here.
